README.de.md

```diff
@@ -5,6 +5,7 @@
 **Deutsch**,
 [English][EN],
 [Español][ES],
+[Italiano][IT],
```

Comment:
Italian should be after French in all of these

Reply:
Fixed